CONTRIBUTING.md

```diff
@@ -39,9 +39,31 @@ Do all your development or fixing work here.
 
 After all your development/fixing work is done, do not forget to add `Unit Test` and `Regression Test` around that. It will be nice if you can add an example of the new feature you have added.
 
+#### * Check your work after running all Unit and Regression Tests
+
+You should run all the unit tests by hitting the following command
+
+```
+mvn clean install
+```
+
+To run regression tests, you need to run a OpenShift/Kubernetes Cluster and after that
```

Comment:
how about `you need to OpenShift/Kubernetes Cluster up and running upfront`? and also some commands(`minikube start` or `minishift start`) or links for how to start cluster using minikube and minishift will really help for user.

Reply:
We are assuming this thing, as kubernetes-client is project based on Openshift/Kubernetes. We already mentioned the requirement. And how to do that is not required for `contribution.md` according to me